config/config.go

```diff
@@ -47,6 +48,21 @@ func New() *Config {
 	return c
 }
 
+// NewFromKiteKey parses the given kite key file and gives a new Config value.
+func NewFromKiteKey(file string) (*Config, error) {
+	key, err := kitekey.ParseFile(file)
+	if err != nil {
+		return nil, err
+	}
+
+	var c Config
```

Comment:
```
var c Config&
c := &Config{}
c := new(Config)
```

It's 1 to 3 keystrokes shorter :sunglasses: :sunglasses: :sunglasses: :trollface: